ui/src/shared/components/TableGraphFormat.tsx

```diff
@@ -64,6 +65,25 @@ interface State {
   invalidDataError: ErrorTypes
 }
 
+const areFormatPropertiesEqual = (prevProperties, newProperties) => {
+  const updatedProps = _.keys(_.omit(prevProperties, 'data')).filter(
+    k => !_.isEqual(newProperties[k], prevProperties[k])
+  )
+
+  return (
+    newProperties.uuid === prevProperties.uuid &&
+    !_.includes(updatedProps, 'tableOptions') &&
```

Comment:
I had a bit of trouble figuring out what this function does. Would the following be clearer? 
I don't know if it's functionally equivalent.

```
const areFormatPropertiesEqual = (prevProperties, newProperties) => {
  const formatProps = [
    'uuid',
    'tableOptions',
    'fieldOptions',
    'timeFormat',
    'sort',
  ]

  const areFormatPropertiesEqual = formatProps.every(k =>
    _.isEqual(prevProperties[k], newProperties[k])
  )

  return areFormatPropertiesEqual
}
```

Reply:
makes sense